data/interfaces/default/base.html

```diff
@@ -20,12 +20,130 @@
     ${next.headIncludes()}
 
     <link rel="icon" type="image/x-icon" href="interfaces/default/images/favicon.ico"/>
-    <!-- touch icons -->
     <link rel="shortcut icon" href="interfaces/default/images/favicon.png">
-    <link rel="apple-touch-icon" href="interfaces/default/images/icon_iphone.png">
-    <link rel="apple-touch-icon" sizes="72x72" href="interfaces/default/images/icon_ipad.png">
-    <link rel="apple-touch-icon" sizes="114x114" href="interfaces/default/images/icon_iphone@2x.png">
-    <link rel="apple-touch-icon" sizes="144x144" href="interfaces/default/images/icon_ipad@2x.png">
+
+    <!-- Allow web app to be run in full-screen mode. -->
+    <meta name="apple-mobile-web-app-capable"
+          content="yes">
+    <!-- Configure the status bar. -->
+    <meta name="apple-mobile-web-app-status-bar-style"
+          content="black">
+    <!-- Set the viewport. -->
+    <meta name="viewport"
+          content="initial-scale=1">
+    <!-- Disable automatic phone number detection. -->
+    <meta name="format-detection"
+          content="telephone=no">
+
+    <!-- ICONS -->
+    <!-- IE10 icon --> 
+    <meta name="PlexPy" content="PlexPy"/>
+    <meta name="msapplication-TileColor" content="#000000"/>
+    <meta name="msapplication-square70x70logo" content="tiny.png"/>
+    <meta name="msapplication-square150x150logo" content="square.png"/>
+    <meta name="msapplication-wide310x150logo" content="wide.png"/>
+    <meta name="msapplication-square310x310logo" content="large.png"/>
+    <!-- Android >M39 icon -->
```

Comment:
Can we use a manifest.json instead? See: https://developers.google.com/web/updates/2015/10/splashscreen?hl=en

Use this:
```
<link rel="manifest" href="interfaces/default/json/Android-manifest.json">
```

Save this file to `interfaces/default/json/Android-manifest.json` and add the referenced icons.
```
{
  "name": "PlexPy",
  "display": "standalone",
  "icons": [
    {
      "src": "../images/res/android/icon-192x192.png",
      "sizes": "192x192",
      "type": "image/png"
    },
    {
      "src": "../images/res/android/icon-256x256.png",
      "sizes": "256x256",
      "type": "image/png"
    },
    {
      "src": "../images/res/android/icon-384x384.png",
      "sizes": "384x384",
      "type": "image/png"
    },
    {
      "src": "../images/res/android/icon-512x512.png",
      "sizes": "512x512",
      "type": "image/png"
    }
  ],
  "background_color": "#1F1F1F",
  "theme_color": "#000000"
}
```